docs/Deploying.md

```diff
@@ -75,23 +75,39 @@ these features.
 ```bash
 java \
     -jar \
-    -Dauthenticate=social_auth
+    -Dauthenticate=[social_auth/social_auth_google],social_auth_microsoft
     portal/target/dependency/webapp-runner.jar \
     portal/target/cbioportal.war
 ```
 
-Only google is supported as optional login currently. One needs to set the
-Google related configuration in the `portal.properties` file:
+Google and Microsoft live are supported as optional login currently. Possible values for authenticate are
+
+```bash
+-Dauthenticate=[social_auth/social_auth_google],social_auth_microsoft
+-Dauthenticate=social_auth/social_auth_google
+-Dauthenticate=social_auth_microsoft
```

Comment:
@kalletlak ☝️ i think this is a bit hard to read. I would remove the `social_auth` option since it's just there for backwards compatibility. That leaves three options: `social_auth_google`, `social_auth_microsoft` or `social_auth_google,social_auth_microsoft`